smarts/benchmark/driving_smarts/v2023/config_3.yaml

```diff
@@ -20,4 +20,4 @@ benchmark:
         - scenarios/sumo/platoon/merge_exit_sumo_t_agents_1
       kwargs:
         seed: 42
-      # metric_formula: smarts/benchmark/driving_smarts/v2023/metric_formula_platoon.py
+      metric_formula: smarts/benchmark/driving_smarts/v2023/metric_formula_platoon.py
```

Comment:
I think we should mention this in the changelog if not already.

Reply:
Previously changelog stated
```
- In the metrics module, the records which is the raw metrics data and the scoring which is the formula to compute the final results are now separated to provided greater flexibility for applying metrics to different environments.
```
Now, another entry, as follows is added.
```
- Benchmark listing may specify specialised metric formula for each benchmark.
```